templates/dashboard/includes/_filters.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+{% load materializecss %}
+
+
+<div class="card">
+  <form method="get">
+    <div class="card-content">
+      <div class="col s12">
+        <span class="card-title">{% trans 'Filters' context 'Dashboard card title' %}</span>
+        <span>{% trans 'Total records found' context 'Dashboard filter' %}: {{ filter.qs|length }}</span>
```

Comment:
Can we use `count` here? I think passing it from the backend would be better, as usually, we are trying to avoid templates firing up additional queries.

Reply:
I will check if this triggers query.

Reply:
It does. In case of a large dataset this would be really slow. Actually we don't display the number of items in queryset in the list views. Maybe it would be better to add it consistently to all lists in a different PR?

Reply:
Hmmm i don't think that another PR is required. I already have to do this so why not now?

Reply:
If we add it to every list then it's fine in this one.

Reply:
Is knowing the precise number of results useful? When is it useful? When is it not? It would be nice to have some use cases before we add something to all list views.

Reply:
For the reason @patrys just mentioned I would split it out to another issue/PR.

Reply:
From the other hand we need a visual feedback that there's some filtering being applied and number of records found partially serve that purpose.